SITCOMTN-076.tex

```diff
@@ -151,7 +151,7 @@ \section{Commissioning Era Information Sharing Policies}
 
 \subsection{Summary}
 
-All pixel images and representations of pixel images of any size field of view, including individual visit images, coadd images, and difference images based on ComCam and LSSTCam commissioning on-sky observations are embargoed for a period of at least 30 days, with the exception of alert postage stamp images that have been publicly released to community alert brokers.
+All pixel images and representations of pixel images of any size field of view, including individual visit images, coadd images, and difference images based on ComCam and LSSTCam commissioning on-sky observations are embargoed for a period of at least 30 days, with the exception of alert postage stamp images that have been publicly released to community alert brokers, and certain specifically approved types of images that illustrate technical performance aspects of the observatory (Section \ref{special_classes}).
```

Comment:
Suggest: "...and certain specifically approved types of images that illustrate technical performance aspects of the observatory (Section \ref{special_classes}) and are not on-sky images."

Reply:
Not adding "are not on-sky images" because the "specifically approved types of images" listed in the draft are images of the sky, but are taken in non-standard modes (donut images for AOS commissioning, pinhole images) or cover an extremely limited field of view (few arcsecond scale) to illustrate some aspect of technical performance. 

Removed a comma for clarity.